src/test/java/org/broadinstitute/hellbender/utils/IntervalUtilsUnitTest.java

```diff
@@ -1795,4 +1795,100 @@ public void testBasicGenomicSortOfLocatableErrorContigNotInDictionary() throws I
                 {list2, gtList2}
         };
     }
+
+    @DataProvider
+    public static Object[][] createIntervalWithStop() {
+        final SAMSequenceDictionary dictionary = SAMSequenceDictionaryExtractor.extractDictionary(new File(FULL_HG19_DICT));
+        return new Object[][]{
+                {dictionary, "1", 100, 200},
+                {dictionary, "2", 780, 900233}
+        };
+    }
+
+    @Test(dataProvider = "createIntervalWithStop")
+    public void testCreateIntervalWithStop(final SAMSequenceDictionary dictionary, final String contig, final int start, final int stop) {
+        Assert.assertEquals(IntervalUtils.createInterval(dictionary, contig, start, stop), new SimpleInterval(contig, start, stop));
+    }
+
+    @DataProvider
+    public static Object[][] createIntervalWithoutStop() {
+        final SAMSequenceDictionary dictionary = SAMSequenceDictionaryExtractor.extractDictionary(new File(FULL_HG19_DICT));
+        return new Object[][]{
+                {dictionary, "1", 100},
+                {dictionary, "2", 780}
+        };
+    }
+
+
+    @Test(dataProvider = "createIntervalWithoutStop")
+    public void testCreateIntervalWithoutStop(final SAMSequenceDictionary dictionary, final String contig, final int start) {
+        Assert.assertEquals(IntervalUtils.createInterval(dictionary, contig, start), new SimpleInterval(contig, start, start));
+    }
+
+    @DataProvider
+    public static Iterator<Object[]> createOverEntireContig() {
+        final SAMSequenceDictionary dictionary = SAMSequenceDictionaryExtractor.extractDictionary(new File(FULL_HG19_DICT));
+        return dictionary.getSequences().stream()
+                .map(seq -> new Object[]{dictionary, seq.getSequenceName(), seq.getSequenceLength()})
+                .iterator();
+    }
+
+
+    @Test(dataProvider = "createOverEntireContig")
+    public void testCreateOverEntireContig(final SAMSequenceDictionary dictionary, final String contig, final int length) {
```

Comment:
add `createOverEntireContig` index tests to this method

Reply:
Already tested through this method, because it delegates on it...